extensions/sql-database-projects/src/test/mainController.test.ts

```diff
@@ -18,6 +18,11 @@ import { shouldThrowSpecificError } from './testUtils';
 
 let testContext: TestContext;
 
+function createController(): MainController {
```

Comment:
Why did you create a separate function for this? It's just a single line as it is so doesn't really seem valuable

Reply:
I was just copying what the ML extension did, but I'll get rid of it